README.md

```diff
@@ -212,6 +212,12 @@ new L.GPX(app.params.gpx_url, {
 }).addTo(map);
 ```
 
+The content of the popup created for waypoints can be customised by setting `marker_options.popupTemplate` to a function returning the popup content, receiving name and description of the waypoint as arguments. The (simplified) default looks like
```

Comment:
Can you wrap to 80 chars like in the rest of the file?

Add a colon at the end of the second sentence, too.

Reply:
Oh. Sorry. I apologize for that and have fixed the issues.

Would you prefer a squashed request? I was unsure :).